src/components/basic/Button.md

```diff
@@ -276,6 +276,36 @@ Html Snippet:
 </a>
 ```
 
+#### Tiny
```

Comment:
Overall looks good. Do we want to add an example of this on a dark bg as well in the markdown file?

Reply:
The reason i didn't is I'm confident it won't display weird on dark backgrounds, since its just smaller?

Reply:
It does. Only reason I would add is just to balance out the default versions of the button section with the dark button section, since one shows three versions of the icon button and the dark version shows two. That's me just looking at the section as a single piece. I think functionally this is fine and not having it as a dark section in the markdown file is not blocking, so this should be good to go.